packages/components/pages/f-self-exclusion/src/components/SelfExclusion.vue

```diff
@@ -1,47 +1,138 @@
 <template>
     <div
+        class="f-self-exclusion"
         :class="$style['c-selfExclusion']"
```

Comment:
Is this intended? There should already be a class of `c-selfExclusion` on this element that can be used for styling?